packages/pn-commons/src/types/index.ts

```diff
@@ -90,4 +91,9 @@ export type {
     SendPaperDetails,
     SideMenuItem,
     Sort,
+    AppResponse,
```

Comment:
Verificare che gli export vengano effettivamente utilizzati all'esterno di pn-commons. In caso contrario rimuoverli

Reply:
ok